src/lib/db.js

```diff
@@ -1,5 +1,17 @@
 const { PrismaClient } = require('@prisma/client')
 
+let prisma = PrismaClient
+
+// This is supposed to improve performance for next.js
```

Comment:
I could be wrong but I don't this this has an effect since we're not running prisma client in our next.js ui code